app/src/main/res/values-ru/sura_names.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <string-array name="sura_names">
+    <item>аль-Фатиха (Открывающая Книгу)</item>
```

Comment:
Where did you get the Sura Names from?
Normally, prefixes are capitalized, but your version are all with lower case.